src/main/resources/com/google/api/codegen/ruby/test.snip

```diff
@@ -222,10 +226,96 @@
   end
 @end
 
+@private serverStreamingTest(test)
+  it 'invokes {@test.clientMethodName} without error' do
+    @if test.hasRequestParameters
+      @# Create request parameters
+      {@initCode(test.initCode)}
+
+    @end
+    @# Create expected grpc response
+    {@initCode(test.mockResponse.initCode)}
+
+    @# Mock Grpc layer
+    {@mockServerStreamingRequest(test)}
+    mock_stub = MockGrpcClientStub.new(:{@test.clientMethodName}, mock_method)
+
+    {@test.grpcStubCallString}.stub(:new, mock_stub) do
+      client = {@test.serviceConstructorName}.new
+
+      @# Call method
+      {@methodCallWithResponse(test)}
+
+      @# Verify the response
+      assert_equal(1, response.count)
+      assert_equal(expected_response, response.first)
+    end
+  end
+@end
+
+@private clientStreamingTest(test)
+  it 'invokes {@test.clientMethodName} without error' do
+    @# Create request parameters
+    {@initCode(test.initCode)}
+    requests = [request]
+    @if test.hasReturnValue
+
+      @# Create expected grpc response
+      {@initCode(test.mockResponse.initCode)}
+    @end
+
+    @# Mock Grpc layer
+    {@mockClientStreamingRequest(test)}
+    mock_stub = MockGrpcClientStub.new(:{@test.clientMethodName}, mock_method)
+
+    {@test.grpcStubCallString}.stub(:new, mock_stub) do
+      client = {@test.serviceConstructorName}.new
+
+      @# Call method
+      {@methodCallWithResponse(test)}
+
+      @# Verify the response
+      @if test.hasReturnValue
+        assert_equal(expected_response, response)
+      @else
+        assert_nil(response)
+      @end
+    end
+  end
+@end
+
+@private bidiStreamingTest(test)
+  it 'invokes {@test.clientMethodName} without error' do
+    @# Create request parameters
+    {@initCode(test.initCode)}
+    requests = [request]
+    @if test.hasReturnValue
+
+      @# Create expected grpc response
+      {@initCode(test.mockResponse.initCode)}
+    @end
+
+    @# Mock Grpc layer
+    {@mockBidiStreamingRequest(test)}
+    mock_stub = MockGrpcClientStub.new(:{@test.clientMethodName}, mock_method)
+
+    {@test.grpcStubCallString}.stub(:new, mock_stub) do
+      client = {@test.serviceConstructorName}.new
+
+      @# Call method
+      {@methodCallWithResponse(test)}
```

Comment:
Ah good catch. I forgot to make a method call for client streaming requests. It has been made and I also made a test case for client streaming calls.